includes/admin/class-wp-job-manager-writepanels.php

```diff
@@ -85,8 +85,8 @@ public function job_listing_fields() {
 				'priority'    => 11,
 				'classes'     => array( 'job-manager-datepicker' ),
 				/* translators: date format placeholder, see https://secure.php.net/date */
-				'placeholder' => _x( 'yyyy-mm-dd', 'Date format placeholder.', 'wp-job-manager' ),
-				'value'       => metadata_exists( 'post', $post->ID, '_job_expires' ) ? get_post_meta( $post->ID, '_job_expires', true ) : calculate_job_expiry( $post->ID ),
+				'placeholder' => metadata_exists( 'post', $post->ID, '_job_expires' ) ? esc_attr_x( 'yyyy-mm-dd', 'Date format placeholder.', 'wp-job-manager' ) : calculate_job_expiry( $post->ID ),
```

Comment:
No need to escape attribute (`_x()` is fine here). Attributes are escaped in templates.